amass/utils/misc_test.go

```diff
@@ -0,0 +1,99 @@
+package utils
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestSubdomainRegex(t *testing.T) {
+	tests := []struct {
+		name     string
+		domain   string
+		event    string
+		expected string
+	}{
+
+		{"subdomain", "owasp.org", "subdomain.owasp.org", "subdomain.owasp.org"},
```

Comment:
The name field in these test tables should conform to a standard format for consistency.

Reply:
Fixed the two tests that did not follow the "Test#: Foo" convention for the Name field.